ui/.gitignore

```diff
@@ -5,6 +5,26 @@
 target/
 *.zip
 installer/vsphere-client-serenity/
+installer/plugin-packages/
+vic-ui-linux
+vic-ui-darwin
+vic-ui-windows.exe
+vic-machine-linux
+vic-ui-automation-test/
+vic-ui-h5c/dev-build.sh
+vic-ui-h5c/prod-build.sh
+vic-ui-h5c/uia/h5-plugin-tests/ui-automation/vic-uia/work/results/
+vic-ui-h5c/uia/h5-plugin-tests/ui-automation/vic-uia/work/runlists/
+vic-ui-h5c/uia/h5-plugin-tests/ui-automation/vic-uia/work/testbeds/
```

Comment:
I think you just need the root folder to be excluded for git ignore to ignore all underneath right?  This could be potentially simplified unless there is a bunch of stuff in vic-ui-h5c/ that needs to be excluded?